ckan/adx_config.ini

```diff
@@ -56,7 +56,7 @@ ckan.datastore.default_fts_index_method = gist
 
 ## Site Settings
 
-ckan.site_url = http://ckan:5000
+ckan.site_url = http://ckan
```

Comment:
With this setup we could start accessing local adr with standard `http://localhost`.

Reply:
@tomeksabala that wouldn't work for me with my ec2 instance setup. So referring to http://ckan is more ideal